uxarray/utils/get_quadratureDG.py

```diff
@@ -1,10 +1,11 @@
 import numpy as np
 from numba import njit, config
+from uxarray.utils.constants import ENABLE_JIT_CACHE, ENABLE_JIT
 
-config.DISABLE_JIT = False
+config.DISABLE_JIT = ENABLE_JIT
```

Comment:
Is there a specific reason why you have the constant named `ENABLE_JIT`, I'd suggest a name like `USE_JIT` for our internal use, since currently `ENABLE_JIT` "enables JIT" when it's set to False.